### DISCUSSION TITLE
Bug Report

### DISCUSSION THREAD

**Discussion by eufex** [apohl79/audiogridder#350] [Help]
I'm running server on windows 7, Plugin on Linux. Ultimately would like to run some NI stuff on the Windows box as it doesn't work in WINE. Just testing with Green Oak Crystal VST with Reaper on the Linux box.
First comment is the plugin is WAY too small on a high res laptop it's like 1cm square on the screen.
I think you also need to mention on the website that the Windows server has dependencies for C++ libraries 2015, 2017, 2019 otherwise users will not be able to get it to work 

Crystal VST loads in the server, you can get the GUI of Crystal up in the plugin in 

Reaper and I've managed to get some audio once but mostly both server and plugin (and Reaper with it) crash.

I hope this is of some help as this is an awesome project.

Server logs and plugins log attached
[AudioGridderServer_2021-02-02_17-41-59.log](https://github.com/apohl79/audiogridder/files/5913453/AudioGridderServer_2021-02-02_17-41-59.log)
[AudioGridderPlugin_2021-02-02_17-47-39.log](https://github.com/apohl79/audiogridder/files/5913492/AudioGridderPlugin_2021-02-02_17-47-39.log)

**Comment by apohl79** [comment]
Regarding the c++ runtime. Thx for this. I updated the build system recently and missed the static linking part for the runtime on windows...

For the UI size, try the zoom factor. You can scale up the plugin UI. Plugin settings menu -> zoom.

Concerning the crashes, please activate tracing. But most importantly see if you get core dumps of the crashes. From the logs alone it is rather hard to analyze the crashes.

Thanks,
Andreas

**Comment by apohl79** [comment]
> the Windows server has dependencies for C++ libraries 2015, 2017, 2019
>> I updated the build system recently and missed the static linking part for the runtime on windows...

Updated the windows build with the runtime statically linked.
